assets/prometheus-k8s/rules.yaml

```diff
@@ -171,6 +171,16 @@ spec:
       labels:
         workload_type: statefulset
       record: mixin_pod_workload
+    - expr: |
```

Comment:
We use jsonnet, so these changes should first go into https://github.com/openshift/cluster-monitoring-operator/blob/master/jsonnet/rules.jsonnet and after commiting you run:
```
make clean
make generate-in-docker
```
and this and other files might be generated, let me know if you have any questions. :)